posixlib/src/main/scala/scala/scalanative/posix/signal.scala

```diff
@@ -0,0 +1,468 @@
+package scala.scalanative
+package posix
+
+import scala.scalanative.native._
+
+/**
+ * Some of the functionality described on this reference page extends the ISO C standard.
+ * Applications shall define the appropriate feature test macro (see XSH The Compilation Environment )
+ * to enable the visibility of these symbols in this header.
+ *
+ * Note 1: The functionality described may be removed in a future version of this volume of POSIX.1-2017
+ * Note 2: The functionality is an option marked as XSI (X/Open System Interfaces)
+ */
+@extern
+object signal {
+  // define the following macros, which shall expand to constant expressions with distinct values
```

Comment:
Is the reference to "macros" here a TODO? Looks like this code does not currently use macros. But could be replaced with a macro?

Reply:
We have to have helper functions in C to support macros currently. The comment is from the standard and that portion is complete.
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/signal.h.html

Reply:
Ah! Makes sense. I see now the reference is to the C preprocessor macros used to define the constant. I was stucking thinking only about Scala macros. LGTM!